tldw_Server_API/app/core/DB_Management/ChaChaNotes_DB.py

```
@@ -32444,6 +31515,96 @@ def get_latest_sync_log_change_id(self) -> int:
             raise
 
 
+def _delegate_conversation_store_method(method_name: str):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_delegate_conversation_store_method`

(ANN202)